qttypes/build.rs

```diff
@@ -150,7 +150,7 @@ fn main() {
         ""
     };
 
-    if cfg!(target_os = "macos") {
+    if cfg!(any(target_os = "macos", target_os = "linux")) {
```

Comment:
actually, i think the target_os is the target on which the build script is compiled,  not the target which the resulting binary should be built for.

And this will not be great for cross compiling either.

Reply:
Indeed, I'll add a commit to this merge request to use an environment variable (I'm sure cargo provides one, will look up :)